CONTRIBUTING.md

```diff
@@ -63,10 +63,11 @@ Even documentation improvements and tests are most welcome:
 1. This PR fixes a README link: https://github.com/qdrant/fastembed/pull/143
 
 ### Adding New Models
-You can start by adding new models to the FastEmbed. You can find all the model requests [here](https://github.com/qdrant/fastembed/labels/model%20request). 
-
-
-There are quite a few pull requests that were merged for this purpose and you can use them as a reference. Here is an example: https://github.com/qdrant/fastembed/pull/129 
+1. Open Requests for New Models are [here](https://github.com/qdrant/fastembed/labels/model%20request). 
+2. There are quite a few pull requests that were merged for this purpose and you can use them as a reference. Here is an example: https://github.com/qdrant/fastembed/pull/129
+3. Make sure to add tests for the new model
+    - The CANONICAL_VECTOR values must come from a reference implementation usually from Huggingface Transformers or Sentence Transformers
```

Comment:
Please describe what's canonical vector. 

Do you mean one has to convert these two 
```
input_texts = [
    "hello world", "flag embedding"
]
```

Using HF Inference API for [the model](https://huggingface.co/intfloat/multilingual-e5-small)?

Reply:
Reference PRs are linked before that sentence